src/components/InfoBanner/blocks/InfoBannerCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { BoxProps, Box } from '../../Box/Box';
+import { Link as WaveLink } from '../../Link/Link';
+import { Text } from '../../Text/Text';
+
+import { ICON_VARIANTS, IconBox, InfoBannerVariants, ROLE_VARIANTS, RoundedBox } from '../InfoBanner';
+
+interface CardProps extends BoxProps {
+    children: ReactNode;
+    variant?: InfoBannerVariants;
+    emphasized?: boolean;
+}
+
+const LinkList = styled.ul`
```

Comment:
I'm wondering if `LinkList` should even exist... I believe the requirement to have a list of links is only for DMT and to my understanding we should only be exposing the components that make the `InfoBanner` (meaning Card, Title, Description and Link).

We'd still be allowing to have a list of links (or whatever the user needs) with the `Link` styled according to our `InfoBanner` specs but the `LinkList` would have to be implemented in DMT, this way we still offer flexibility without coupling Wave with DMT.

Otherwise if a different project needs a list of descriptions wouldn't we have to create a `DescriptionList` even though the `InfoBanner` doesn't really use it?

Reply:
I agree with Martí, I think this should be a custom DMT thing

Reply:
ok, I removed it 👍